### DISCUSSION TITLE
Webdriver for form filling automation ?

### DISCUSSION THREAD

**Discussion by ooralas** [webdriverio/webdriverio#9995] [Community]
I have a project where I need to automate filling out different forms on external websites. When I search online, I always find results that talk about Webdriverio only for UI testing automation, but my project is not about testing at all.In my project, there are no special requirements. Depending on the form, certain script will be executed.
Therefore, I would like to ask if Webdriverio is suitable for my purpose as a framework, or should I rather use Selenium or another framework?

I know that UI frameworks can be used for automation, but what made me wonder and ask my question here is that I found only rare results that explain how Webdriverio can be used for automation.

**Comment by christian-bromann** [comment]
WebdriverIO can totally used for your use case. It is primarily used for testing UI , yes but that doesn't mean you can not use it for other use cases too.

**Reply by ooralas** [reply]
Perfect. Thanks for your answer. :) 👍🏻 

@christian-bromann apart from being Head of WDIO Project 😁, what do you think, would you choose Selenium or WDIO in this case ?

**Reply by christian-bromann** [reply]
Both tools help you achieve your use case. Give both a test ride and decide based on your experience.

**Reply by ooralas** [reply]
I don't have a lot of time to test the both of them in all aspects, therefore I am looking for a summarized answer. But thanks for your answer :).
